resources/js/package.js

```diff
@@ -56,7 +56,7 @@ function init() {
     window.app = new Vue({
         el: '#app',
         data: {
-            custom: {},
+            custom: { openAgreement: null },
```

Comment:
This is for custom stuff people build, not for the core itself. Can we do this without this?